pyproject.toml

```diff
@@ -90,7 +90,7 @@ colorhash = "~1.0.2"
 jsonschema = "~3.2"
 packaging = ">=20.0,<21.0"
 pytz = ">=2019.1,<2021.0"
-rasa-sdk = "^2.2.0"
+rasa-sdk = "^2.3.0"
```

Comment:
2.3.1 ?

Reply:
Mhm, pip should find out that `uvloop` isn't available on 2.3.0 and automatically install 2.3.1 on Windows systems, no?

Reply:
One can hope so 😅  alright, I'll trust you on this, you know more about deps than me lol

Reply:
I'll bump it. Better be safe than sorry.